### ISSUE TITLE
Declare minimal OGC ontologies in OGC-NA register

### ISSUE THREAD

**Issue by sgrellet** [opengeospatial/SELFIE#67]
The three minimal ontologies (HY_Feature, GSML basic, GWML2) should be 'loaded/declared' within OGC-NA register for us to start using them for real in SELFIE (and raise new issues :) ).
Ideally 
- with an 'experimental' status (like https://raw.githubusercontent.com/ukgovld/registry-core/master/src/main/vocabs/registryVocab.ttl#statusExperimental) and not 'proposed (http://www.opengis.net/def/status/proposed) as no formal SWG work was done
- asking for ttl should directly go to the files as proposed by Abdel.

As per this discussion (https://github.com/opengeospatial/HY_Features/pull/274) what could be done to support  @rob-metalinkage on this ?

**Comment by rob-metalinkage** [comment]
Hi folks
first the great news -  next month i start on a 3 year 0.5 FTE contract with OGC working on related matters - so supporting SELFIE gets that much easier.  

I have set up infrastructure so we have an option for "experimental" to be either using www.opengis.net or defs-dev.opengis.net namespaces.  I think we're probably OK to push resources to www.  but I will be tweaking server behaviour on defs-dev to conform to a new W3C API recommendation [1]  (which reduces burden on me documenting behaviour too!) 

I'll have a look at the complete hyf ontology as soon as i can and get back to y'all

1. https://www.w3.org/TR/dx-prof-conneg/

**Comment by dblodgett-usgs** [comment]
This is great, Rob. Let's go ahead and get these into `www.opengis.net` as experimental if that's kosher with @ghobona. In parallel the relevant SWGs will work on motions to publish the experimental schemas. 

My suggestion would be for @rob-metalinkage to have a look at the three ontologies linked here: https://github.com/opengeospatial/SELFIE/issues/50#issuecomment-540042468 and either  
1. create follow on issues for yourself to get them published, 
1. register issues here in SELFIE to fix things, or 
1. just go ahead and publish them as experimental.

**Comment by ghobona** [comment]
Sounds good!

**Comment by dblodgett-usgs** [comment]
@KathiSchleidt -- any progress on this topic at the IE this week?

**Comment by afeliachi** [comment]
I've been to the geosemantics session in Toulouse and I had the occasion to ask the question about publishing domain **OWL** ontologies in the OGC server. Linda van den Brink told me that we should contact the geosemantics list about the procedure to follow, but reassured me that if they're "valid", there would be no problem adding them.

**Comment by ghobona** [comment]
@afeliachi I can confirm that if the ontologies are valid they can be published. To speed up the process, when you are ready, please ask the GeoSemantics DWG to review and approve the ontologies. A portal vote or e-mail vote would be fine. The OGC Naming Authority will rely on the advice or recommendation of the GeoSemantics DWG.

**Comment by afeliachi** [comment]
@ghobona thank you. Will do that :).

@dblodgett-usgs I'll try to finish reviewing the ontologies content. Some [minor editions](https://github.com/opengeospatial/GeoSciML/issues/27) are necessary after the discussion we had during the SELFIE impromptu session in Toulouse TC. I'll need your confirmation for HYF also.

**Comment by dblodgett-usgs** [comment]
Interesting, @ghobona that is different from the guidance you gave previously (https://github.com/opengeospatial/SELFIE/issues/67#issuecomment-543442029). I thought the decision we made was to have these published as "experimental" and pass these to the relevant SWGs to have them voted on and approved.

**Comment by ghobona** [comment]
@dblodgett-usgs I don't see this as different from previous advice because a DWG is supposed to review outputs of IP initiatives. The 'experimental' route does not mean that there is no review. The IE may proceed with using the 'experimental' route, however at some point a DWG (in this case, the GeoSemantics DWG) will have to review the ontologies.

**Comment by dblodgett-usgs** [comment]
Right -- I would agree if these were _new_ ontologies. But these Ontologies are minimal OWL ontologies based on UML of existing specifications from active SWGs.

**Comment by KathiSchleidt** [comment]
Sorry for the silence, but was sick all last week, most I could manage was a bit of remote participation in Toulouse
To the conundrum above, how about the following approach:

- as soon as @afeliachi  can manage a semi-stable draft of the three minimal ontologies (HY_Feature, GSML basic, GWML2), he provides them to @rob-metalinkage  for provision via defs-dev.opengis.net namespaces
- the GeoSemantics DWG can review and comment/approve this resource at their leisure, hopefully leading to eventual publication at www.opengis.net, while we get on with things

Thoughts?

**Comment by ghobona** [comment]
I'm happy with the approach proposed by @KathiSchleidt.

**Comment by rob-metalinkage** [comment]
IMHO these should go to the final URL and have its status marked
appropriately...

e.g.
https://www.opengis.net/def/appschema/hy_features/hyf/HY_CatchmentAggregate

so we have hyf as the namespace for HY_Hydrofeature - as per xml namespace
and equivalent models

when it comes to content defs-dev is for things we dont have a governance
model defined for, or are new types of things we are playing with, or
examples to illustrate behaviour.
(we could create additional places to keep content separate)

We need a bundle of all the artefacts needed for each application schema so
we can put it in places that the different views resolve correctly:
1) RDF/OWL file
2) XMI file
3) XSD schema (note I still need to work out how to systematically link to
objects managed in schema.opengis.net where the URL paths are not
predictable ;-( )

The other thing I want to do is to formally define a profile of UML,  and
one for the shapechange conversion configuration - so we can declare what
forms the resulting OWL is - given there are possible options - i need the
BRGM wizards to define this and provide the documents needed for me to
follow this for other application schema (or pass on to others) - happy to
contribute to a whitepaper or methodology paper - its important this
process is transparent and repeatable for all schemas - e.g. GWML. A formal
profile like this proves a Linked Data way of discovering all the relevant
documentation.


Rob

**Comment by dblodgett-usgs** [comment]
Yes -- as @rob-metalinkage says, we should publish to the opengis.net urls but label things experimental. Let's try and focus on minimum viable to get to a place that URIs resolve. I see and appreciate the need for additional work to get to reproducibility, but we need to keep this work as narrowly focused (incremental) as we can.

**Comment by dblodgett-usgs** [comment]
We have content ready here:  
https://github.com/opengeospatial/HY_Features/blob/master/ontology/minimal/hyf_minimal.ttl
and once this PR is merged here:  
https://github.com/opengeospatial/GeoSciML/pull/31

@denevers Can you merge that PR not that @afeliachi added the missing types? 

@KathiSchleidt I think this is ready to move forward.

**Comment by dblodgett-usgs** [comment]
Without technical leadership, this issue can not move forward. Happy for someone to reopen and try and get this over the finish line, but closing for now as can't fix.

**Comment by afeliachi** [comment]
I'll make sure  the needed files will be delivered, will close again this issue afterwards
